supabase/migrations/20241121201709_aritst_social_links_table.sql

```diff
@@ -1,6 +1,7 @@
 create type "public"."social_type" as enum ('TIKTOK', 'YOUTUBE', 'INSTAGRAM', 'TWITTER', 'SPOTIFY', 'APPLE');
 
-drop table "public"."artists";
+-- FIX: Drop the artists table if it exists (from README.md documented solution)
+drop table if exists "public"."artists";

```

Comment:
typically, we delete this line.

question: what happens if we keep this line in and ship to main? 🤔

Reply:
as it is deleting table, will create no issue if not any table it in ref.